pages/common/openssl-dgst.md

```diff
@@ -0,0 +1,24 @@
+# openssl dgst
+
+> OpenSSL command to generate digest values and perform signature operations.
+> More information: <https://www.openssl.org/docs/man1.0.2/man1/dgst.html>.
+
+- Generate a SHA256 hash value of a file:
```

Comment:
Based on `sha256sum` page:
```suggestion
- Calculate the SHA256 checksum for a file, saving the result to a specific file:
```

Reply:
I don't like the term *checksum* here, because it is usually used only for non-cryptographic algorithms. Would *digest* be OK as well?

Reply:
Sure